espflash/src/bin/espflash.rs

```diff
@@ -196,6 +202,23 @@ fn erase_region(args: EraseRegionArgs, config: &Config) -> Result<()> {
     Ok(())
 }
 
+fn reset(args: ConnectArgs, config: &Config) -> Result<()> {
+    let mut args = args.clone();
+    args.no_stub = true;
+    let mut flash = connect(&args, config)?;
+    info!("Resetting target device");
+    flash.connection().reset()?;
+
+    Ok(())
+}
+
+fn hold_in_reset(args: ConnectArgs, config: &Config) -> Result<()> {
```

Comment:
Thank you for naming this function helpfully!  I have to remind folks that the default state for an MCU is to be running, and that is has to be held in reset (or powered down) to NOT run.  So when we "reset" the MCU, we are actually "releasing from reset".  This function name makes that very explicit.